kotlin/lib/src/main/kotlin/SvixOptions.kt

```diff
@@ -1,6 +1,6 @@
 package com.svix.kotlin
 
-data class SvixOptions(val debugUrl: String = DEFAULT_URL) {
+data class SvixOptions(val serverUrl: String = DEFAULT_URL) {
```

Comment:
you need to use the new serverUrl in all the clients now

Reply:
Oops, fixed.